apps/crn-server/src/migrations/1681729305-enforce-url-format.ts

```diff
@@ -0,0 +1,238 @@
+/* istanbul ignore file */
+import {
+  RestEvent,
+  RestNews,
+  RestPage,
+  RestTeam,
+  RestUser,
+} from '@asap-hub/squidex';
+import { Migration } from '@asap-hub/server-common';
+import { applyToAllItemsInCollection } from '../utils/migrations';
+
+export default class EnforceUrlFormat extends Migration {
+  isPatchingEnabled = false;
+
+  cleanupUrl = (url?: string) => {
```

Comment:
maybe extract this function to a helper function and test it?

I would also probably extend it with the http/https adding logic, so in the end the function would take any string, trim, conver to lowercase and add http or https.

Reply:
As discussed we'll merge as it is as we need the two step implementation